### ISSUE TITLE
UI: async job refreshes to a screen not currently viewed

### ISSUE THREAD

**Issue by DaanHoogland** [apache/cloudstack#5172]
After deleting a vm and going into the details view of another VM, the screen is refreshed on another view.

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
main
~~~

##### STEPS TO REPRODUCE

1. Delete a vm (with expunge and disks)
1. Go into the details view of another VM
1. When deleting the first VM is done, the screen refreshes into the VM overview list, abandoning the current details view.

##### EXPECTED RESULTS

~~~
The details view is still visible
~~~

##### ACTUAL RESULTS

~~~
The overview is shown
~~~

**Comment by nvazquez** [comment]
@davidjumani can you please check this issue?

**Comment by davidjumani** [comment]
This is a code wide issue and might take some time, I'll look into it

**Comment by davidjumani** [comment]
Will be easier to fix after https://github.com/apache/cloudstack/pull/4782 is merged

**Comment by davidjumani** [comment]
Closing as issue is fixed in #5214
